docs/snippets/Snippets.NUnit/Attributes/TestFixtureAttributeExamples.cs

```diff
@@ -37,6 +39,17 @@ public ParameterizedTestFixture(int eq1, int eq2, int neq)
             _neq = neq.ToString();
         }
 
+        // Can use params arguments (but not yet optional arguments)
+        public ParameterizedTestFixture(params char[] eqArguments)
+        {
+            _eq1 = eqArguments[0].ToString();
+            _eq2 = eqArguments[1].ToString();
+            if (eqArguments.Length > 2)
+                _neq = eqArguments[2].ToString();
+            else 
+                _neq = null;
```

Comment:
`_neq` declaration needs updating, append `?`, to allow assigning `null`

Reply:
Fixing